lib/ansible/modules/files/acl.py

```diff
@@ -73,6 +73,14 @@
     type: bool
     default: 'no'
     version_added: "2.0"
+
+  recalculate_mask:
+    description:
+      - Select if and when to recalculate the effective right masks of the files, see setfacl documentation for more info. Incompatible with C(state=query).
+    choices: [ default, mask, no_mask ]
```

Comment:
tempted to have no default and make it 'yes|no' boolean, use 'default' behaviour when `None`

Reply:
I think that could be confusing, because sometimes the rights masks do get recalculated when neither '--mask' nor '--no-mask' are used. 
What about changing the choices to 'default', 'always', 'never' and explaining in the description that 'always' maps to '--mask' and 'never' maps to '--no-mask'?